safety_limiter/src/safety_limiter.cpp

```diff
@@ -330,9 +333,45 @@ class SafetyLimiterNode
   }
   double predict(const geometry_msgs::Twist& in)
   {
+    if (cloud_accum_->size() == 0)
+    {
+      if (allow_empty_cloud_)
+      {
+        return 1.0;
+      }
+      ROS_WARN_THROTTLE(1.0, "safety_limiter: Empty pointcloud passed.");
+      return 0.0;
+    }
+
+    try
+    {
+      geometry_msgs::TransformStamped odom_to_base = tfbuf_.lookupTransform(
+          base_frame_id_,
+          cloud_accum_->header.frame_id,
+          pcl_conversions::fromPCL(cloud_accum_->header.stamp),
+          ros::Duration(0.1));
```

Comment:
My main concern when I implemented this node was this delay.
Ideal implementation would be:
1. lookupTransfrom with `timeout = 0`
2. If not looked-up, lookupTransform with `stamp = 0` and extrapolate transform.

Reply:
thanks, I'll fix it to concern about your advice.

Reply:
addressed